src/include/migraphx/streamutils.hpp

```diff
@@ -59,6 +60,12 @@ inline stream_range_container<Range> stream_range(const Range& r)
 
 namespace detail {
 
+inline void
+stream_write_value_impl(rank<3>, std::ostream& os, const migraphx::value::binary& binary_object)
+{
+    os << binary_object;
+}
```

Comment:
This can be removed, and also the value.hpp include as well.

Reply:
I have it because, otherwise it would match with `rank<2>` method instead.

Reply:
removed and changed the overloading and ranking.